neurom/fst/tests/test_neuritefunc.py

```diff
@@ -213,6 +213,12 @@ def test_section_branch_orders():
                     (0, 1, 1,  # type 3, basal dendrite
                      0, 1, 1)) # type 2, axon
 
+def test_section_strahler_orders():
+    strahler_orders = list(_nf.section_strahler_orders(SIMPLE))
+    assert_allclose(strahler_orders,
+                    (2, 1, 1,  # type 3, basal dendrite
```

Comment:
After a discussion with @mgeplf we decided to use the Stahler order as defined here: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3928584/pdf/1755-7682-7-6.pdf 
I will modify the code accordingly and include this reference.